tests/ssg_test_suite/test_env.py

```diff
@@ -248,40 +227,152 @@ def reset_state_to(self, state_name, new_running_state_name):
 
         return new_container
 
-    def _find_image_by_name(self, image_name):
-        return self.client.images.get(image_name)
+    def _delete_saved_state(self, image):
+        self._terminate_current_running_container_if_applicable()
+
+        assert self.created_images
+
+        associated_image = self.created_images.pop()
+        assert associated_image == image
+        self._remove_image(associated_image)
+
+    def offline_scan(self, args, verbose_path):
+        command_list = self._local_oscap_check_base_arguments() + args
+
+        return common.run_cmd_local(command_list, verbose_path)
+
+    def _commit(self, container, image):
+        pass
```

Comment:
I would prefer to have `NotImplementedError` here, as this is basically an Abstract Base Class.

Reply:
OK, I'll change that.

Reply:
Changed.